internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/shares.go

```diff
@@ -61,10 +61,11 @@ type Handler struct {
 	publicURL              string
 	sharePrefix            string
 	homeNamespace          string
-	resourceInfoCacheTTL   time.Duration
 	additionalInfoTemplate *template.Template
 	userIdentifierCache    *ttlcache.Cache
 	resourceInfoCache      gcache.Cache
+	resourceInfoCacheTTL   time.Duration
+	useResourceInfoCache   bool
```

Comment:
Should we use it in the user-configurable struct? If a user sets `useResourceInfoCache` to true but doesn't set `resourceInfoCacheTTL` assuming a default value, we won't use the cache. I'd suggest removing this and checking  the value of `resourceInfoCacheTTL` in all places

Reply:
Okay, no it's not user configurable. But I'd still vote for removing the extra variable. Doesn't really add any value

Reply:
It does add a bit of readability but I have no hard opinion on this, I can remove it again.